### PR TITLE
Rocky8 changes

### PR DESCRIPTION
**Pull request by amanda11**
Changes for rockylinux8 - part of https://github.com/StackStorm/community/issues/88

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,11 @@ below.
 VirtualBox Extension packs ([follow instructions for Extension packs
 here](https://www.virtualbox.org/manual/ch01.html#intro-installing)).
 
+* Note. Problems have been found using VirtualBox 5.1 with RockyLinux 8, which were resolved
+using version 6.1. 
+If using VirtualBox 6.1.28 or above, the default allowed range is 192.168.56.0/21.
+Therefore, you may want to set VM_IP or alter the default range as the default IP used is 192.168.16.20.
```
Comment (armab) [inline README.md:25]: Can we modify it in the `Vagrantfile` instead to make the defaults safe?

Comment (amanda11) [reply README.md]: > Can we modify the `Vagrantfile` directly to make the defaults safe?

Updated

**Review by arm4b — APPROVED** [review]
Thanks for updating it!

Just left one more minor comment.

**Review by cognifloyd — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -19,6 +19,9 @@ below.
 VirtualBox Extension packs ([follow instructions for Extension packs
 here](https://www.virtualbox.org/manual/ch01.html#intro-installing)).
 
+* Note. Problems have been found using VirtualBox 5.1 with RockyLinux 8, which were resolved
+using version 6.1.
```
[the remark this answers is not in the document]
Comment (armab) [reply README.md:23]: I understand now.
Makes sense, thanks for more context :+1:

**Pull request merged**